bindings/node/README.md

```diff
@@ -0,0 +1,158 @@
+MongoDB Client Encryption
+=========================
+
+The node.js wrapper for [`libmongocrypt`](../../README.md)
+
+### Requirements
+
+Follow the instructions for building `libmongocrypt` [here](../../README.md#building-libmongocrypt).
+
+### Installation
+
+Now you can install `mongodb-client-encryption` with the following:
+
+```bash
+npm install mongodb-client-encryption
+```
+
+### Testing
+
+Run the test suite using:
+
+```bash
+npm test
+```
+
+# Documentation
+
+## Classes
+
+<dl>
+<dt><a href="#AutoEncrypter">AutoEncrypter</a></dt>
+<dd><p>An internal class to be used by the driver for auto encryption
+<strong>NOTE</strong>: Not meant to be instantiated directly, this is for internal use only.</p>
+</dd>
+<dt><a href="#ClientEncryption">ClientEncryption</a></dt>
+<dd><p>The public interface for explicit client side encryption</p>
+</dd>
+</dl>
+
+<a name="AutoEncrypter"></a>
+
+## AutoEncrypter
+An internal class to be used by the driver for auto encryption
+**NOTE**: Not meant to be instantiated directly, this is for internal use only.
+
+
+* [AutoEncrypter](#AutoEncrypter)
+
+    * [new AutoEncrypter(options)](#new_AutoEncrypter_new)
+
+    * [.encrypt(ns, cmd, callback)](#AutoEncrypter+encrypt)
+
+    * [.decrypt(buffer, callback)](#AutoEncrypter+decrypt)
+
+
+<a name="new_AutoEncrypter_new"></a>
+
+### new AutoEncrypter(options)
+
+| Param | Type | Description |
+| --- | --- | --- |
+| options | <code>object</code> | Optional settings |
+| options.client | <code>MongoClient</code> | The parent client auto encryption is enabled on |
+| options.mongocryptdClient | <code>MongoClient</code> | The client used for communication with `mongocryptd` |
+| options.keyVaultNamespace | <code>string</code> | The namespace of the key vault, used to store encryption keys |
+| options.schemaMap | <code>object</code> |  |
+| options.kmsProviders | <code>object</code> |  |
+| options.logger | <code>function</code> |  |
+
+Create an AutoEncrypter
+
+<a name="AutoEncrypter+encrypt"></a>
+
+### *autoEncrypter*.encrypt(ns, cmd, callback)
+
+| Param | Type | Description |
+| --- | --- | --- |
+| ns | <code>string</code> | The namespace for this encryption context |
+| cmd | <code>object</code> | The command to encrypt |
+| callback | <code>function</code> |  |
+
+Encrypt a command for a given namespace
+
+<a name="AutoEncrypter+decrypt"></a>
+
+### *autoEncrypter*.decrypt(buffer, callback)
+
+| Param | Type |
+| --- | --- |
+| buffer | <code>\*</code> | 
+| callback | <code>\*</code> | 
```

Comment:
Does this need to be `<code>function</code>`?

Reply:
yes, but this is auto-generated. We have a separate issue for providing accurate API documentation [here](https://jira.mongodb.org/browse/NODE-2055).